recipes/numina/meta.yaml

```diff
@@ -0,0 +1,130 @@
+{% set name = "numina" %}
+{% set version = "0.16" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "2ce685760054a00bc34cd4d1fbef2359f7b17981a490286a67911432e9c7679f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - numina = numina.user.cli:main
+    - numina-bpm = numina.array.bpm:main
+    - numina-imath = numina.tools.imath:main
+    - numina-wavecalib = numina.array.wavecalib.__main__:main
+    - numina-ximshow = numina.array.display.ximshow:main
+    - numina-ximplotxy = numina.array.display.ximplotxy:main
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pip
+    - toolchain
+    - numpy 1.11.*
+  host:
+    - python
+    - setuptools
+    - six >=1.7
+    - numpy >=1.11
+    - astropy >=1.3
+    - scipy
+    - pyyaml
+    - matplotlib
+    - enum34;python_version <"3.4"
```

Comment:
For the `enum34`, you're going to need to use the appropriate selector:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#preprocessing-selectors